lib/go-tc/deliveryservices.go

```diff
@@ -1608,7 +1611,8 @@ func (ds DeliveryServiceV5) Downgrade() DeliveryServiceV4 {
 			},
 			TLSVersions: make([]string, len(ds.TLSVersions)),
 		},
-		Regional: ds.Regional,
+		Regional:             ds.Regional,
+		RequiredCapabilities: ds.RequiredCapabilities,
```

Comment:
This line is unnecessary since it's being set in all cases below - also possibly confusing because it's the only shallow copy in a sea of deep copies. Or, you could set this to `make([]string, len(ds.RequiredCapabilities))` here, and then below you *only* need to handle the case where `len(ds.RequiredCapabilities)` is greater than zero.